mne/viz/epochs.py

```diff
@@ -96,6 +102,19 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     scale_vmax = True if vmax is None else False
     vmin, vmax = _setup_vmin_vmax(data, vmin, vmax)
 
+    if overlay_times is not None and len(overlay_times) != len(data):
+        raise ValueError('size of overlay_times parameter (%s) do not '
+                         'match the number of epochs (%s).'
```

Comment:
hmm ... okay. Not sure why I see this then in the warning:

```py
mne/viz/epochs.py:116: UserWarning: Some values in overlay_times fall outside of the epochs time interval (between -0.0998976065792 s and 0.299692819738 s)
  (epochs.tmin, epochs.tmax))
```
is it expected to see `epochs.tmin` and `epochs.tmax` in the warning?